src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -121,7 +122,9 @@ def setup_cdk_params(self, config: dict) -> dict:
         else:
             suffix = self.manifest.build.id + '-' + self.manifest.build.architecture
         return {
-            "distributionUrl": self.manifest.build.location,
+            "distributionUrl": self.manifest.build.location if isinstance(self.manifest, BundleManifest) else
+            f"https://artifacts.opensearch.org/snapshots/core/opensearch/{self.manifest.build.version}/opensearch-min-"
+            f"{self.manifest.build.version}-linux-{self.manifest.build.architecture}-latest.tar.gz",
```

Comment:
Not really liking this but does build manifest not providing any of this information?
If so I would seriously thinking about Prudhvi suggestion to just assemble raw min without plugin before pushing to artifact.opensearch.org.

Reply:
Probably create an issue related to this as a later improvements?

Reply:
I think we are already doing that https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L156-L168

Just need to upload the manifest too?

Reply:
As of now the BuildManifest doesn't have `location` field in the manifest therefore had to format the url to download the min distribution. 
I will create an issue to track this and update once the field becomes available.

Reply:
I believe what we are saying is today snapshots `min` is created same way as we build distribution but just with 1 component `OpenSearch`. So it has all the things that distribution does. Regarding commit id that you want, we can get that if we also upload the manifest.yml to artifacts.opensearch.org in the same way as we upload min tarball.

Reply:
See code here:
https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L156-L168
and 
uploading part here:
https://github.com/opensearch-project/opensearch-build-libraries/blob/main/vars/uploadMinSnapshotsToS3.groovy#L43-L59

Reply:
@gaiksaya Peter's change uploads the min-snapshot manifest too, it is of type BuildManifest which doesn't have `location` field in it as per schema.  
min-snapshot manifest https://artifacts.opensearch.org/snapshots/core/opensearch/2.9.0-SNAPSHOT/opensearch-min-2.9.0-SNAPSHOT-linux-arm64-latest.tar.gz.build-manifest.yml
@peterzhuamazon

Reply:
@gaiksaya we are creating with build, not assemble after build, this means only build manifest is present which does not have the location information.

By running another round of assemble without plugin would just repackage min into a tar with a startup script and bundle manifest. The reason I am hesitate because that startup script will break due to no SEC and PA plugins.

Therefore, I would say put this as a later improvement in an issue.

Reply:
Bundling min with startup script is not the right way forward as it will fail due to lack of security plugin.

Reply:
I see what you mean! Yeah we need to add check if plugin exists in start up scripts.

Reply:
Created issue https://github.com/opensearch-project/opensearch-build/issues/3686 to track above mentioned requirements. @gaiksaya @peterzhuamazon